app/workers/deletes.php

```diff
@@ -95,6 +121,60 @@ protected function deleteUser(Document $document, $projectId)
         ], $this->getProjectDB($projectId));
     }
 
+    protected function deleteExecutionLogs() 
+    {
+        $this->deleteForProjectIds(function($projectId) {
+            if (!($projectDB = $this->getProjectDB($projectId))) {
+                throw new Exception('Failed to get projectDB for project '.$projectId);
+            }
+
+            // Delete Executions
+            $this->deleteByGroup([
+                '$collection='.Database::SYSTEM_COLLECTION_EXECUTIONS,
+                '$projectId='.$projectId
```

Comment:
Execution logs don't have the projectId attribute. You can check system collections structures here: `app/config/collections.php`. This filter will always return zero documents. Because Appwrite has separate tables for each projects you can just delete all execution logs with not special filters.

Reply:
Okay understood 👍